src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -3,33 +3,73 @@ import React, { FC } from "react"
 import styled from "styled-components"
 import { createFragmentContainer, graphql } from "react-relay"
 import { Details_conversation } from "v2/__generated__/Details_conversation.graphql"
+import { Media } from "v2/Utils/Responsive"
 
-const BorderedFlex = styled(Flex)`
+export const DETAIL_BOX_ANIMATION = `transition: width 0.2s ease-in-out;`
```

Comment:
`ease-out` feels snappier and a bit more elegant

Reply:
Could we try `transition: all 300ms cubic-bezier(0.4, 0, 0.2, 1) 0s;` @sepans

Reply:
👍 How does it look now @willanderson?

**xl:**

![xl-conv-cub](https://user-images.githubusercontent.com/687513/83895934-01890780-a719-11ea-8724-e64c633fcdc6.gif)

---
**xs:**

![xs-conv-cub](https://user-images.githubusercontent.com/687513/83895949-06e65200-a719-11ea-985f-7f57f715f96a.gif)

Reply:
Ohh thats much better